superset/config.py

```diff
@@ -308,6 +308,8 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     "TAGGING_SYSTEM": False,
     "SQLLAB_BACKEND_PERSISTENCE": False,
     "LISTVIEWS_DEFAULT_CARD_VIEW": False,
+    "DISPLAY_MARKDOWN_HTML": True,
+    "ESCAPE_MARKDOWN_HTML": False,
```

Comment:
```suggestion

    # When set to True, this flag escapes/shows HTML (rather than rendering it) in Markdown components
    "ESCAPE_MARKDOWN_HTML": False,
```